### ISSUE TITLE
Doesn't work with prettier-atom

### ISSUE THREAD

**Issue by marcofugaro** [observablehq/prettier-react-tachyons#3]
Hey, first congrats on this project, it's awesome!

I wanted to try it with [prettier-atom](https://atom.io/packages/prettier-atom) since it runs prettier on the file save, but it's not working, I'm getting this error even after having installed the dependency.

![image](https://user-images.githubusercontent.com/7217420/38021899-597032a0-327e-11e8-9a93-3c19c0694806.png)

Any way it could work with the atom editor?

**Comment by tmcw** [comment]
My first guess would be - does your project have prettier as a dependency, or are you relying on the copy of prettier bundled with prettier-atom? If prettier isn't a dependency of the project, the path `./node_modules...` will be likely be resolved relative to the atom plugin, rather than the project's root.

**Comment by marcofugaro** [comment]
@tmcw yes, the project has a .prettierrc, prettier as a devDependency, and a prettier script in the package.json's scripts

**Comment by tmcw** [comment]
Try using an absolute path to the configuration instead of a relative one?

**Comment by marcofugaro** [comment]
Hey it works that way!

Sadly we can't commit that to the project's `.prettierrc`, it would be great it every editor could read from it.
